common/fixation_entry.go

```diff
@@ -26,7 +26,9 @@ import (
 //    - GetEntry(index, *entry): get a copy (and reference) of the latest version of an entry
 //    - PutEntry(index, block): drop a reference of a version of an entry
 //    - [TBD] RemoveEntry(index): mark an entry as unavailable for new GetEntry() calls
-//    - GetAllEntryIndex(): get all the entries indices (without versions)
+//    - GetAllEntryIndices(): get all the entries indices (without versions)
```

Comment:
do we need a module migration here?

Reply:
not needed - just another API that leverages the existing data format.